demos/embeddings/deep-graph-infomax-cora.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<ul><li>Instead of "squeeze batch dim out" maybe the comment could be adjusted to something like "for full batch models, squeeze out the batch dim (which is 1)". (I guess this also applies to the comment in the GCN version above.)</li><li>Why a batch size of 2700 for GraphSAGE? I think that results in two batches, one of size 2700 and one of size 8? Maybe we could go with either a less unusual number (like 1000), or definitely the full graph (<code>G.number_of_nodes()</code> )?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>